### PR TITLE
frontend improvements

### PR DESCRIPTION
**Pull request by Meskat**
- decreased header size
- created icon cell components
- swapped type, direction, status cells with proper icons
- combined messages and transactions count columns on smaller screens
- removed public key column on mobile screens
-removed block_id and message on the mobile screen in the transactions table
-formatted date with moment js (time column)
-used zebra styled rows in tables
- remove scale effect on hover on mobiles (used darker bg on hover for readability)
- created tests for icon cells components

### CODE REVIEW

**Review by c3zi — CHANGES_REQUESTED** [review]

**Review by c3zi — CHANGES_REQUESTED** [review]
Please check

File: package.json [hunk 1]
```diff
@@ -120,11 +120,14 @@
       "!app/components/organisms/Breadcrumbs/*.{js,jsx}",
       "!app/config.js",
 
+      "!app/components/**/Button.js",
```
[the remark this answers is not in the document]
Comment (Meskat) [reply package.json]: cause it's not tested yet (and probably even doesn't need to cause its pack of styled components) and it broked tests coverage

File: app/components/molecules/Table/IconCells/StatusTableCell.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    </IconTableCell>
+  );
+};
+
+StatusTableCell.propTypes = {
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+    PropTypes.element,
+  ]).isRequired,
```
[the remark this answers is not in the document]
Comment (Meskat) [reply app/components/molecules/Table/IconCells/StatusTableCell.js]: I've tried it but I've got an error ` Failed prop type: Invalid prop `value` of type `object` supplied to `StatusTableCell`, expected `number`.`

File: package.json [hunk 3]
```diff
@@ -120,11 +120,14 @@
...
       "!app/components/**/messages.js",
       "!app/components/**/styled.js",
       "!app/components/**/Loadable.js",
 
       "!app/containers/*/messages.js",
+      "!app/containers/AppWrapper",
+      "!app/containers/TransactionsListPage/index.js",
```
[the remark this answers is not in the document]
Comment (Meskat) [reply package.json]: coverage

File: app/components/molecules/Table/IconCells/StatusTableCell.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Meskat) [reply app/components/molecules/Table/IconCells/StatusTableCell.js]: I've tried it but I've got an error ` Failed prop type: Invalid prop `value` of type `object` supplied to `StatusTableCell`, expected `number`.`

**Review by c3zi — APPROVED** [review]

**Pull request merged**